test/test_dependencies.jl

```diff
@@ -0,0 +1,47 @@
+using Mimi
+
+#list of URLs of branches of packages to test
+dependencies = ["https://github.com/davidanthoff/fund.jl/archive/master.zip"]
+
+#list of failed tests to build as you go
+errors = []
+
+#make a temporary directory to run the tests in
+run(`mkdir tmp_testing`)
+cd("tmp_testing")
```

Comment:
Ideally we would not rely on the current path anywhere, but instead use absolute paths everywhere that are inferred from the location of this file. Essentially the idea would be to use something like this ``tmp_path = joinpath(dirname(@__FILE__),"tmp_testing")``. Not urgent, we can discuss tomorrow.